config/settings.yml

```diff
@@ -1081,6 +1081,10 @@
   :timeout: 120
   :use_vim_broker: true
   :authentication_timeout: 30.seconds
+  :remote_miq_api:
+    :user: admin
```

Comment:
Can we just leave it nil and raise an error if it's not set and someone attempts to execute a remote power operation?